scheduler/cloudformation/cloudformation.go

```diff
@@ -192,6 +193,23 @@ func (s *Scheduler) SubmitWithOptions(ctx context.Context, app *scheduler.App, s
 	return tx.Commit()
 }
 
+func (s *Scheduler) Restart(ctx context.Context, app *scheduler.App, ss scheduler.StatusStream) error {
+	stackName, err := s.stackName(app.ID)
+	if err != nil {
+		return err
+	}
+	output := make(chan stackOperationOutput)
+	return s.updateStack(ctx, &updateStackInput{
```

Comment:
We may want to do the same thing that we do in `Submit` and wait if `ss != nil`. Otherwise, a consumer that provides a StatusStream, doesn't really know when the Restart has finished.